src/tools/pub/automated-publishing.md

```diff
@@ -316,7 +316,7 @@ grant Cloud Build permission to impersonate this service account.
 1. [Create a cloud project][12], if you don't have an existing project.
 1. Create a _service account_ as follows:
 
-    ```sh
+    ```terminal
     gcloud iam service-accounts create pub-dev \
```

Comment:
To get proper highlighting for the entries changed in this file, you'll need to add the prompt symbol (`$`) to each.